core/bounds.gms

```diff
@@ -437,162 +405,128 @@ vm_capEarlyReti.up(t,regi,"biochp") = 1;
+  vm_deltaCap.fx(t,regi,te,rlf) $ (rlf.val > 1) = 0;
+  vm_cap.fx(ttot,regi,te,rlf) $ (rlf.val > 1) = 0;
+);
 
-v_shGreenH2.lo(t,regi)$(t.val eq 2025) = c_shGreenH2 * 2/3;
-v_shGreenH2.lo(t,regi)$(t.val gt 2025) = c_shGreenH2;
 
-***----------------------------------------------------------------------------
-*** upper bound on bioliquids as a share of transport liquids
-***----------------------------------------------------------------------------
+*** lower bound on share of green hydrogen starting from 2030 (switch c_shGreenH2 has default value zero)
+v_shGreenH2.lo(t,regi) $ (t.val = 2025) = c_shGreenH2 * 2/3;
+v_shGreenH2.lo(t,regi) $ (t.val > 2025) = c_shGreenH2;
 
-v_shBioTrans.up(t,regi)$(t.val > 2020) = c_shBioTrans;
 
-***----------------------------------------------------------------------------
-*** bounds on final energy use (relevant in case some switches are acitvated that make pm_shfe_up and pm_shfe_lo non-zero)
-***----------------------------------------------------------------------------
+*** Limit slack variable and uncontrolled variable values for adj costs that limit changes to reference in cm_startyear
+v_changeProdStartyearSlack.up(t,regi,te) $ ( (t.val > 2005) and (t.val = cm_startyear) ) = + c_SlackMultiplier * p_adj_seed_reg(t,regi) * p_adj_seed_te(t,regi,te) ;
+v_changeProdStartyearSlack.lo(t,regi,te) $ ( (t.val > 2005) and (t.val = cm_startyear) ) = - c_SlackMultiplier * p_adj_seed_reg(t,regi) * p_adj_seed_te(t,regi,te) ;
 
-*** upper and lower bounds on FE carrier shares
-v_shfe.up(t,regi,entyFe,sector)$pm_shfe_up(t,regi,entyFe,sector) = pm_shfe_up(t,regi,entyFe,sector);
-v_shfe.lo(t,regi,entyFe,sector)$pm_shfe_lo(t,regi,entyFe,sector) = pm_shfe_lo(t,regi,entyFe,sector);
 
-*** upper and lower bounds on gases+liquids share in FE
-v_shGasLiq_fe.up(t,regi,sector)$pm_shGasLiq_fe_up(t,regi,sector) = pm_shGasLiq_fe_up(t,regi,sector);
-v_shGasLiq_fe.lo(t,regi,sector)$pm_shGasLiq_fe_lo(t,regi,sector) = pm_shGasLiq_fe_lo(t,regi,sector);
-
-*** Set H2 upper bound in buildings for years defined at cm_H2InBuildOnlyAfter
-vm_demFeSector.up(t,regi,"seh2","feh2s","build",emiMkt)$(t.val le cm_H2InBuildOnlyAfter) = 1e-6;
+*cb 20120319 avoid negative adjustment costs in 2005 (they would allow the model to artificially save money)
+v_adjFactor.fx("2005",regi,te) = 0;
 
-***----------------------------------------------------------------------------
-*'  Limit slack variable and uncontrolled variable values for adj costs that limit changes to reference in cm_startyear
-***----------------------------------------------------------------------------
 
-v_changeProdStartyearSlack.up(t,regi,te)$( (t.val gt 2005) AND (t.val eq cm_startyear ) ) = + c_SlackMultiplier * p_adj_seed_reg(t,regi) * p_adj_seed_te(t,regi,te) ;
-v_changeProdStartyearSlack.lo(t,regi,te)$( (t.val gt 2005) AND (t.val eq cm_startyear ) ) = - c_SlackMultiplier * p_adj_seed_reg(t,regi) * p_adj_seed_te(t,regi,te) ;
+*** fix F-gas emissions to inputdata (IMAGE)
```

Comment:
What about making an extra section "emissions"? We also have this section in core/declarations now and we plan to create a separate emissions module.